lms/djangoapps/ora_staff_grader/tests/test_views.py

```diff
@@ -72,65 +80,83 @@ class TestInitializeView(BaseViewTest):
     """
     Tests for the /initialize view, creating setup data for ESG
     """
-    view_name = 'ora-staff-grader:initialize'
+
+    view_name = "ora-staff-grader:initialize"
 
     def setUp(self):
         super().setUp()
         self.log_in()
 
-    @ddt.data({}, {PARAM_ORA_LOCATION: ''})
+    @ddt.data({}, {PARAM_ORA_LOCATION: ""})
     def test_missing_param(self, query_params):
-        """ Missing ORA location param should return 400 and error message """
+        """Missing ORA location param should return 400 and error message"""
         response = self.client.get(self.api_url, query_params)
 
         assert response.status_code == 400
         assert json.loads(response.content) == {"error": ERR_MISSING_PARAM}
 
     def test_bad_ora_location(self):
-        """ Bad ORA location should return a 400 and error message """
-        response = self.client.get(self.api_url, {PARAM_ORA_LOCATION: 'not_a_real_location'})
+        """Bad ORA location should return a 400 and error message"""
+        response = self.client.get(
+            self.api_url, {PARAM_ORA_LOCATION: "not_a_real_location"}
+        )
 
         assert response.status_code == 400
         assert json.loads(response.content) == {"error": ERR_BAD_ORA_LOCATION}
 
-    @patch('lms.djangoapps.ora_staff_grader.views.get_submissions')
-    @patch('lms.djangoapps.ora_staff_grader.views.get_course_overview_or_none')
+    @patch("lms.djangoapps.ora_staff_grader.views.get_submissions")
+    @patch("lms.djangoapps.ora_staff_grader.views.get_course_overview_or_none")
     def test_init(self, mock_get_course_overview, mock_get_submissions):
-        """ Any failure to fetch info returns an unknown error response """
+        """Any failure to fetch info returns an unknown error response"""
         mock_course_overview = CourseOverviewFactory.create()
         mock_get_course_overview.return_value = mock_course_overview
         mock_get_submissions.return_value = test_data.example_submission_list
 
-        response = self.client.get(self.api_url, {PARAM_ORA_LOCATION: self.ora_usage_key})
+        response = self.client.get(
+            self.api_url, {PARAM_ORA_LOCATION: self.ora_usage_key}
+        )
 
-        expected_keys = set(['courseMetadata', 'oraMetadata', 'submissions'])
+        expected_keys = set(["courseMetadata", "oraMetadata", "submissions"])
         assert response.status_code == 200
         assert response.data.keys() == expected_keys
 
-    @patch('lms.djangoapps.ora_staff_grader.views.get_submissions')
-    @patch('lms.djangoapps.ora_staff_grader.views.get_course_overview_or_none')
-    def test_init_xblock_exception(self, mock_get_course_overview, mock_get_submissions):
```

Comment:
What's the line length config configured in your linter? This line's only 90, i don't see why it would be broken

Reply:
Black defaults to 88, which is shorter than our enforced 120 but, again, I just ran the script.